phylo/src/evolutionary_models/mod.rs

```diff
@@ -41,6 +43,22 @@ impl Display for DNAModelType {
     }
 }
 
+impl DNAModelType {
+    pub fn get_model_type(model_name: &str) -> Self {
```

Comment:
nit: Please consider naming it `from`, or better use https://doc.rust-lang.org/std/convert/trait.From.html

Reply:
Done.